docker-compose.yml

```diff
@@ -1,11 +1,17 @@
 version: "3"
 
 services:
+  # replace with normal redis?
```

Comment:
Not sure if we want to do this at all or if it should be a different MR

Reply:
We can do it here as it's used only for local development.

Reply:
I've removed the comment, can't figure out why it's not working with the normal redis image